pkgs/misc/emulators/duckstation/default.nix

```diff
@@ -53,6 +53,7 @@ mkDerivation rec {
   ];
 
   cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
You should set `cmakeBuildType` instead if you want to set this, the CMake setup hook will use it accordingly & it's easier to override. As you can see though, it already defaults to `Release` so this isn't needed.

https://github.com/NixOS/nixpkgs/blob/c7c159af4d542580fd62c98a61eb0efd885dce74/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L107

```suggestion
```